app/views/annotations/_annotations_overview_accordion.html.erb

```diff
@@ -0,0 +1,72 @@
+<% accordionLabel = "annotations-overview-accordion" %>
+<% accordionLabel = "student-#{accordionLabel}" if is_students_annotations %>
+<div class="accordion" id="<%= accordionLabel %>">
+
+  <% annotations_by_lecture.each_with_index do |(lecture, annotations), index| %>
+    <div class="accordion-item">
+      <% collapseLabel="annotation-overview-collapse-#{index}-#{is_students_annotations}" %>
+
+        <!-- Accordion header -->
+        <h2 class="accordion-header">
+          <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse"
+              data-bs-target="#<%= collapseLabel %>"
+              aria-expanded="true" aria-controls="#<%= collapseLabel %>">
+            <%= lecture %>
+          </button>
+        </h2>
+
+        <!-- Accordion body -->
+        <div class="accordion-collapse collapse" id="<%= collapseLabel %>"
+            data-bs-parent="#<%= accordionLabel %>">
+          <div class="accordion-body">
+
+            <% annotations.each do |annotation| %>
+              <a href="<%= annotation[:link] %>" style="text-decoration: none;">
```

Comment:
It might be worth thinking about whether we want the link to the video open in a new tab: This would be uniform with the behaviour of the "Play" button on media cards.